CMakeLists.txt

```diff
@@ -17,20 +18,36 @@ option(DOWNLOAD_ENSMALLEN "If ensmallen is not found, download it." ON)
 option(DOWNLOAD_STB_IMAGE "Download stb_image for image loading." ON)
 option(BUILD_GO_SHLIB "Build Go shared library." OFF)
 option(BUILD_DOCS "Build doxygen documentation (if doxygen is available)." ON)
+set(BOARD_NAME "" CACHE INTERNAL "Enter the board name to compile mlpack for.")
+
+if(CMAKE_CROSSCOMPILING)
+  set(GFORTRAN_LIBRARY "" CACHE STRING "Path for gfortran library.")
+  set(PTHREAD_LIBRARY "" CACHE STRING "Path for pthread library")
```

Comment:
Actually I am not sure it is necessary to have these lines here.  If we call `find_library( ... )` later to set the value of `GFORTRAN_LIBRARY`, then that will create the cache entry.

Reply:
Exactly, there is no need for these ones here